dev/README.md

```diff
@@ -10,98 +10,104 @@ Currently successfully brings up charts - no guarantee that everything is workin
 - kubectl
 
 ## Regtest
-
-run in the `dev` folder:
-```
-direnv allow
-make create-cluster
-make init
-make deploy-services
-make deploy
-```
+* run in the `dev` folder:
+  ```
+  direnv allow
+  make create-cluster
+  make init
+  make deploy-services
+  make deploy
+  ```
 
 ### Test
 
-Forward the galoy API port:
-```
-kubectl -n galoy-dev-ingress port-forward svc/ingress-nginx-controller 8080:443
-```
-In an other terminal:
-```
-curl -k 'https://localhost:8080/graphql' -H 'Content-Type: application/json' -H 'Accept: application/json' --data-binary '{"query":"mutation login($input: UserLoginInput!) { userLogin(input: $input) { authToken } }","variables":{"input":{"phone":"+59981730222","code":"111111"}}}'
-```
-
-Expected result:
-```
-{"data":{"userLogin":{"authToken":"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1aWQiOiI2MzM1NDA5MTgzZmZmNTFiYWUyMjE1OWQiLCJuZXR3b3JrIjoicmVndGVzdCIsImlhdCI6MTY2NDQzNDMyMX0.Dc6M49I6TQfqS0ZlmIMrwu71GdCcDDzwZsyTb-EVyMk"}}}
-```
-
-Currently incomplete functionality - but depending on what you want to hack on it'll work.
+#### Forward the oathkeeper-proxy port:
+* Run and keep open:
+  ```
+  kubectl -n galoy-dev-galoy port-forward svc/galoy-oathkeeper-proxy 8080:4455
```

Comment:
is there a way to port forward the ingress instead? for the graphql api to work properly beyond the user login it needs some of the nginx headers `x-real-ip` and `x-forwarded-for` for rate limiting. 

currently i can port forward `443` the ingress like this:
`kubectl -n galoy-dev-ingress port-forward --address 0.0.0.0 svc/ingress-nginx-controller 8080:443`

is it possible to port forward port `80`? i am having cors issues when trying to connect the web wallet to the grapphql backend since they are running on HTTP vs HTTPS.

when i try to port forward the ingress on 80 like this:
`kubectl -n galoy-dev-ingress port-forward --address 0.0.0.0 svc/ingress-nginx-controller 8080:80` the browser just redirects to https://localhost instead of staying on http://localhost:8080. Any ideas?

Reply:
In the dev setup ingress-nginx is not activated for kratos (hence it is set to be connected on `galoy-kratos-public:80`)
so we so far testing directly through the `oathkeeper-proxy` rather than the `ingress-nginx-controller`.

To test with ingress on (which would be more similar to the production environment) we would need to activate nginx all over the stack and use the `https:// (:443) on the kratos and api endpoints.

Reply:
Found that we can pass the ingress annotation:
`nginx.ingress.kubernetes.io/ssl-redirect: "false"`
for the dev env api and that should help the above issue
Can be done in a separate PR.
The ingress for the admin api is configured in a similar way in: https://github.com/GaloyMoney/charts/pull/2511

Reply:
@openoms  if i wanted to test/hack this real quick in in my local dev chars where would i set this?

Reply:
put it under:
```
  annotations:
    nginx.ingress.kubernetes.io/ssl-redirect: "false"
```
in `charts/galoy/templates/api-ingress.yaml`

For the PR we can make this configurable and only passed for the dev env.

Reply:
in thus PR keeping the readme to suggest going through oathkeeper for now as the admin/graphql endpoint only work through that until #2511

Reply:
yes, if we can make that configurable that would be awesome!

Reply:
it worked! this is exciting!!! i got the web wallet working in dev chars (without any weird proxies, *had to do some minor tweaks in some other charts) 

1. forward the api on 80=>4002 `kubectl -n galoy-dev-ingress port-forward --address 0.0.0.0 svc/ingress-nginx-controller 4002:80`
2. forward the web-wallet on 3000=>3000 `kubectl -n galoy-dev-addons port-forward --address 0.0.0.0 web-wallet-mobile-86447545cd-z8xnf 3000:3000`
3. open http://localhost:3000

I am binding to `0.0.0.0` so i can potentially ngrok my web wallet on local dev charts out to the public

Reply:
would it help you to add this configurable:
`nginx.ingress.kubernetes.io/ssl-redirect: "false"`
annotation to this PR? 
and can also rebase to contain the oathkeeper changes from main

Reply:
done in [e6af906](https://github.com/GaloyMoney/charts/pull/2516/commits/e6af9060f0db16db832e286e41ca38171b1248ca) so could test on top this branch